irc/config.go

```diff
@@ -723,6 +723,14 @@ type Config struct {
 		} `yaml:"tagmsg-storage"`
 	}
 
+	Metadata struct {
+		// BeforeConnect       int `yaml:"before-connect"` todo: this
+		Enabled       bool
```

Comment:
Disable the cap in supportedCaps if `Enabled` is false, compare:

https://github.com/ergochat/ergo/blob/0f5603eca2485d783742fa17c77226e56d8bc6eb/irc/config.go#L1612-L1614